docs/device_api.md

```diff
@@ -441,7 +487,7 @@ In the case of complex commands requiring parameters, the `commandValue` could b
 }
 ```
 
-This example will tell the device to reply to a ping message with the provided params. Note that `=` cannot be used instead of `:` given that `=` is [a forbidden character for Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html), so the update at CB triggering the command would be never progressed.
+This example will tell the device to reply to a ping message with the provided params. Note that `=` cannot be used instead of `:` given that `=` is [a forbidden character for Context Broker] (https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html), so the update at CB triggering the command would be never progressed.
```

Comment:
Same here

Reply:
OK. To be corrected in the next pull request.